openstef/pipeline/create_component_forecast.py

```diff
@@ -2,26 +2,71 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
+import joblib
 import pandas as pd
 import structlog
-
 import openstef.postprocessing.postprocessing as postprocessing
 from openstef.data_classes.prediction_job import PredictionJobDataClass
+from openstef.model.regressors.dazls import Dazls
+from openstef.enums import ForecastType
+from openstef import PROJECT_ROOT
+
+# Set the path for the Dazls stored model
+DAZLS_STORED = PROJECT_ROOT / "openstef" / "data" / "dazls_stored.sav"
+
+
+def create_input(pj, input_data, weather_data):
+    """
+    This function prepares the input data, which will be used for the Dazls model prediction, so they will be
+    according Dazls model requirements.
+
+    :param pj: pj (PredictionJobDataClass): Prediction job
+    :param input_data: (pd.DataFrame): Input forecast for the components forecast.
+    :param weather_data: (pd.DataFrame): Weather data with 'radiation' and 'windspeed_100m' columns
+
+    :return: input_df (pd.Dataframe): It outputs a dataframe which will be used for the Dazls prediction function.
+    """
+
+    # Prepare raw input data
+    input_df = weather_data[["radiation", "windspeed_100m"]].merge(
+        input_data[["forecast"]].rename(columns={"forecast": "total_substation"}),
+        how="inner",
```

Comment:
Your code looks quite readable to me, eventhough I am new to the repository. Good job on that!

Considering this merge I was wondering why you chose to do an "inner"-merge, as opposed to a right merge for example? Does there ever occur a situation in which you would want rows from the right dataframe to be excluded after the merge?